claster.py

```diff
@@ -17,31 +17,30 @@
         Covid_death.append(int(row['COVID-19 Deaths']))
         State.append(row['State'])
 
-States = set(State)
+States = list(set(State))
+state_to_num = {States[i]: i for i in range(len(States))}
 Covid_deaths = [0]*len(States)
 Deaths = [0]*len(States)
-i = 0
 
-for states in States:
-    for state in State:
-        if(state == states):
-            Covid_deaths[i] += Covid_death[i]
-            Deaths[i] += Death[i]
-    i += 1
```

Comment:
Works for O(len(States)*len(State)) time.
Uses for summing up state deaths only 50 or so counties (len(Covid_death) is around 800, but program utilizes about 50 its values) because of using same "i" for both arrays of vastly different length.